fixtures/flash-professional/led-par-64-slim-7x10w-rgbw-mk2.json

```diff
@@ -0,0 +1,595 @@
+          "type": "ColorPreset",
+          "comment": "Light Magenta",
+          "colors": ["#ff80ff"],
+          "switchChannels": {
+            "Speed": "Pulse Speed"
+          }
+        },
+        {
+          "dmxRange": [76, 80],
+          "type": "ColorPreset",
+          "comment": "Light Yellow",
+          "colors": ["#ffff80"],
+          "switchChannels": {
+            "Speed": "Pulse Speed"
+          }
+        },
+        {
+          "dmxRange": [81, 85],
+          "type": "ColorPreset",
+          "comment": "White",
+          "colors": ["#ffffff"],
+          "switchChannels": {
+            "Speed": "Pulse Speed"
+          }
+        },
+        {
+          "dmxRange": [86, 90],
+          "type": "ColorPreset",
+          "comment": "White",
+          "colors": ["#ffffff"],
+          "colorTemperature": "9000K",
+          "switchChannels": {
+            "Speed": "Pulse Speed"
+          }
+        },
+        {
+          "dmxRange": [91, 95],
+          "type": "ColorPreset",
+          "comment": "White",
+          "colors": ["#ffffe0"],
+          "colorTemperature": "6500K",
+          "switchChannels": {
+            "Speed": "Pulse Speed"
+          }
+        },
+        {
+          "dmxRange": [96, 100],
+          "type": "ColorPreset",
+          "comment": "White",
+          "colors": ["#feffbe"],
+          "colorTemperature": "5600K",
+          "switchChannels": {
+            "Speed": "Pulse Speed"
+          }
+        },
+        {
+          "dmxRange": [101, 105],
+          "type": "ColorPreset",
+          "comment": "White",
+          "colors": ["#ffffb4"],
+          "colorTemperature": "5000K",
+          "switchChannels": {
+            "Speed": "Pulse Speed"
+          }
+        },
+        {
+          "dmxRange": [106, 110],
+          "type": "ColorPreset",
+          "comment": "White",
+          "colors": ["#feff8f"],
+          "colorTemperature": "4500K",
+          "switchChannels": {
+            "Speed": "Pulse Speed"
+          }
+        },
+        {
+          "dmxRange": [111, 115],
+          "type": "ColorPreset",
+          "comment": "White",
+          "colors": ["#ffdf8b"],
+          "colorTemperature": "3200K",
+          "switchChannels": {
+            "Speed": "Pulse Speed"
+          }
+        },
+        {
+          "dmxRange": [116, 120],
+          "type": "ColorPreset",
+          "comment": "White",
+          "colors": ["#ffcb70"],
+          "colorTemperature": "2500K",
+          "switchChannels": {
+            "Speed": "Pulse Speed"
+          }
+        },
+        {
+          "dmxRange": [121, 125],
+          "type": "ColorPreset",
+          "comment": "White",
+          "colors": ["#ff9f31"],
+          "colorTemperature": "2200K",
+          "switchChannels": {
+            "Speed": "Pulse Speed"
+          }
+        },
+        {
+          "dmxRange": [126, 130],
+          "type": "Effect",
+          "effectName": "Macro 1",
```

Comment:
These macros are ColorJump or ColorFade effects, so better use the preset and make "Macro <n>" a comment.